### PR TITLE
adjust graph events to latest format

### PR DESCRIPTION
**Pull request by anhou**
1. adjust graph events to latest event format
2. fix formats and contents about node and heartbeat routing key
3. fix some indents

### CODE REVIEW

**Comment by panpan0000** [comment]
test this please

**Comment by anhou** [comment]
test this please

**Comment by changev** [comment]
test this please

File: lib/protocol/events.js [hunk 1]
```diff
@@ -273,28 +282,68 @@ function eventsProtocolFactory (
...
-        var routingKey = data.type + '.' + data.typeId + '.' + data.action +
-                         '.' + data.severity + '.' + data.nodeId;
+    EventsProtocol.prototype.publishExternalEvent = function (eventData) {
+        assert.object(eventData);
+        assert.string(eventData.type);
+        assert.string(eventData.action);
+        assert.string(eventData.severity);
+        assert.string(eventData.typeId);
+        assert.ok(eventData.hasOwnProperty("data"));
+
```
Comment (iceiilin) [inline lib/protocol/events.js]: suggest to add hint for err msg here, like 
```
assert.string(eventData.typeId, 'typeId');
```
In this case, if the parameter doesn't contain typeId, the error msg would be "typeId (string) is required".
In current situation, the error msg is "undefined (string) is required", which is difficult for debugging.

**Comment by anhou** [comment]
test this please

**Comment by yyscamper** [comment]
+1

**Pull request merged**